src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,29 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Login method was called with login: {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
+            logger.error("Authentication failed for user: {}", login);
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User {} successfully authenticated", login);
         return user;
     }
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
         // this user identifier should be set by DB. We will use dummy data for this example
         user.setUserId(2L);
+        logger.debug("User found with login: {}", login);
```

Comment:
The log message in the 'findByLogin' method should be more informative. Include information about the action being performed and the parameters received.